code/modules/mob/living/simple_animal/simple_animal_vr.dm

```diff
@@ -49,6 +50,8 @@
 // Simple animals have only one belly.  This creates it (if it isn't already set up)
 /mob/living/simple_animal/init_vore()
 	vore_init = TRUE
+	if(holodeck)	//hacky dumbshit, but I can't get it to work right anyway
```

Comment:
If you applied my suggestion below, holodeck spawned mobs should now properly possess the HOLOGRAM_1 flag, which means you can replace this new redunant var, check and cynical remark with:
```suggestion
	if(CHECK_BITFIELD(flags_1, HOLOGRAM_1))
```